src/main/java/com/google/devtools/build/lib/rules/cpp/CcCompilationHelper.java

```diff
@@ -800,6 +800,11 @@ public CompilationInfo compile() throws RuleErrorException {
     outputGroupsBuilder.putAll(
         CcCommon.createSaveFeatureStateArtifacts(
             cppConfiguration, featureConfiguration, ruleContext));
+    outputGroupsBuilder.put(
+        "index_store",
```

Comment:
`objc_index_store` ?

Reply:
Changed to `cc_index_store` as this is also valid for cc rules